cmd/gitbase-playground/main.go

```diff
@@ -27,7 +27,7 @@ type appConfig struct {
 	Host      string `envconfig:"HOST" default:"0.0.0.0"`
 	Port      int    `envconfig:"PORT" default:"8080"`
 	ServerURL string `envconfig:"SERVER_URL"`
-	DBConn    string `envconfig:"DB_CONNECTION" default:"root@tcp(localhost:3306)/none?maxAllowedPacket=4194304"`
+	DBConn    string `envconfig:"DB_CONNECTION" default:"gitbase@tcp(localhost:3306)/none?maxAllowedPacket=4194304"`
```

Comment:
Is this working out of the box? do you need any extra steps to configure gitbase to accept this user?

Reply:
that's the `gitbase` default https://github.com/src-d/gitbase/blob/master/Dockerfile#L18 and it worked out of the box with that username (with `root` failed)

Reply:
Indeed, this is very confusing.

It would be nice to add a small notice to "Running from Source" section of contributing.md about this - that in case of running gitbase from source, this ENV bar becomes a must. 

That would save some frustration for people who read the docs.

Reply:
I was thinking in purposing the refactor of the `DB_CONNECTION` parameter to be:
```shell
DB_USER default:"gitbase"
DB_PASSWORD
DB_HOST default:"localhost"
DB_PORT default:"3306"
```

That way it would be easy to documment each one, and to modify only some parts of the connection string.

But I thought it would be better to handle this suggestion in a separated issue.
Would you agree on discussing it in the next iteration? If you do, I'll open an issue with your suggestions and my ideas.